.gitmodules

```diff
@@ -16,3 +16,9 @@
 [submodule "minidx9"]
 	path = minidx9
 	url = https://github.com/hrydgard/minidx9.git
+[submodule "common"]
+	path = common
+	url = https://github.com/RPCS3/common
+[submodule "rsx_program_decompiler"]
```

Comment:
rsx_program_decompiler is usable tool for
1) test and fix shaders decompilation
2) extract shaders(/resources) from elf64
3) extract ucode from shader
much easier change code in one place than migrate changes between projects